variables.tf

```diff
@@ -168,6 +168,16 @@ variable "worker_sg_ingress_from_port" {
   default     = "1025"
 }
 
+variable "workers_addtional_policies" {
```

Comment:
typo: `workers_addtional_policies` instead of `workers_additional_policies`

Reply:
fixed